### DISCUSSION TITLE
How do I close a form?

### DISCUSSION THREAD

**Discussion by OldestCoderInTown** [twinbasic/twinbasic#1324] [General]
Form1.Show (vbModal)
             ClientsPage.Visible = False
                          
This brings form1 back tot the front, but will not make clientspage close.

In VB I would expect me.hide to do the trick.

**Comment by dmrvb** [comment]
Showing and closing of forms is all wrong, they show in a strange "semi-modal" way and don't close as expected. I think a user clicking the control box is the only way to close a form. Initial very limited form support was provided a while ago and needs more work. I hope and believe that these things are on Waynes "todo" list.

**Reply by WaynePhillipsEA** [reply]
Yes, very much known and will be re-worked soon.

**Comment by WaynePhillipsEA** [comment]
Only modal forms are supported at the moment.   You currently use Form.Close (e.g. Me.Close) to close the current modal form.

**Reply by fafalone** [reply]
What's the difference between Me.Close and Unload Me?
